python/ray/tests/test_cli.py

```diff
@@ -85,7 +85,7 @@ def configure_lang():
 def cleanup_ray():
     """Shutdown all ray instances"""
     yield
-    runner = CliRunner()
+    runner = CliRunner(env={"RAY_USAGE_STATS_PROMPT_ENABLED": "0"})
```

Comment:
I've bulk-added these as otherwise some of the tests were failing. Maybe we can clean this up in a follow-up